--- origsrc/pango-1.20.5/configure.in	2008-07-02 10:27:50.000000000 -0500
+++ src/pango-1.20.5/configure.in	2008-07-06 19:35:03.187500000 -0500
@@ -622,14 +622,9 @@
 # Note that -module isn't included here since automake needs to see it to know
 # that something like pango-arabic-fc.la is a valid libtool archive
 #
-LIBRARY_LIBTOOL_OPTIONS="-version-info $VERSION_INFO"
-MODULE_LIBTOOL_OPTIONS="-export-dynamic -avoid-version"
-if test "$pango_os_win32" = yes; then
-    # We currently use .def files on Windows
-    true
-    LIBRARY_LIBTOOL_OPTIONS="$LIBRARY_LIBTOOL_OPTIONS -no-undefined"
-    MODULE_LIBTOOL_OPTIONS="$MODULE_LIBTOOL_OPTIONS -no-undefined"
-else
+LIBRARY_LIBTOOL_OPTIONS="-version-info $VERSION_INFO -no-undefined"
+MODULE_LIBTOOL_OPTIONS="-export-dynamic -avoid-version -no-undefined"
+if test "$pango_os_win32" != yes; then
     # libtool option to control which symbols are exported
     # right now, symbols starting with '_' are not exported
     LIBRARY_LIBTOOL_OPTIONS="$LIBRARY_LIBTOOL_OPTIONS "'-export-symbols-regex "^pango_.*"'
