--- origsrc/pango-1.24.0/pango-view/Makefile.am	2009-03-16 16:25:28.000000000 -0500
+++ src/pango-1.24.0/pango-view/Makefile.am	2009-03-24 23:00:42.926500000 -0500
@@ -91,7 +91,7 @@ dist_man_MANS = pango-view.1
 $(srcdir)/pango-view.1: ../configure.in $(pango_view_SOURCES)
 	$(MAKE) $(AM_MAKEFLAGS) pango-view
 	@echo "Creating $@ using help2man(1)"
-	@$(top_builddir)/missing --run \
+	@$(top_srcdir)/missing --run \
 	   help2man --no-info --section=1 \
 		    --help-option="--help-all" --output="$@.tmp" \
 		    --name 'Pango text viewer' ./pango-view \
